charts/wharf-helm/README.md

```diff
@@ -756,66 +756,17 @@ helm install my-release iver-wharf/wharf-helm
 
 ### `web.oidc`
 
-> Settings for the OpenId Connect authentication login.
+> Settings for the OpenId Connect authentication login. These are the config values for the angular-auth-oidc-client NPM package that wharf-web uses. The `web.oidc.redirectUrl` and `web.oidc.postLogoutRedirectUri` defaults to the `global.url` Helm value prefixed with `https://`. See documentation on configs here: https://nice-hill-002425310.azurestaticapps.net/docs/documentation/configuration#config-values See source code for configs here: https://github.com/damienbod/angular-auth-oidc-client/blob/release_13_1_0/projects/angular-auth-oidc-client/src/lib/config/openid-configuration.ts Note: These configs are ignored unless `web.oidcEnabled` is set to `true`.
 
 *Type:* `object`\
-*Default:* `{"authority":"https://login.microsoftonline.com/841df554-ef9d-48b1-bc6e-44cf8543a8fc/v2.0/.well-known/openid-configuration","clientId":"01fcb3dc-7a2b-4b1c-a7d6-d7033089c779","ignoreNonceAfterRefresh":true,"logLevel":"debug","maxIdTokenIatOffsetAllowedInSeconds":600,"scope":"openid profile email offline_access api://wharf-internal/read api://wharf-internal/admin api://wharf-internal/deploy","silentRenew":true,"useRefreshToken":true}`
+*Default:* `{"authority":"https://login.microsoftonline.com/841df554-ef9d-48b1-bc6e-44cf8543a8fc/v2.0/.well-known/openid-configuration","autoUserInfo":false,"clientId":"01fcb3dc-7a2b-4b1c-a7d6-d7033089c779","ignoreNonceAfterRefresh":true,"issValidationOff":false,"logLevel":2,"maxIdTokenIatOffsetAllowedInSeconds":600,"postLogoutRedirectUri":null,"redirectUrl":null,"responseType":"id_token token","scope":"openid profile email offline_access api://wharf-internal/read api://wharf-internal/admin api://wharf-internal/deploy","silentRenew":true,"useRefreshToken":true}`
```

Comment:
These two area very hard to read in raw and when rendered, could be nice to get some linebreaks and maybe codeblock+pretty JSON for the default value?

Reply:
Yea I thought about that too, though the README.md is autogenerated based on the values.yaml file, so I have little control over this.

This line is rendered by this in the template: 

https://github.com/iver-wharf/wharf-helm/blob/4d8a4bd402be303f1326d9e46bb6629dfcaa30f1/charts/wharf-helm/README.md.gotmpl#L67-L74

Would be nice to for example make all "object" types be rendered as code blocks instead of inline code spans, but looking into it there's some problems:

- We only get a string value inside the template of the value, pre-encoded as JSON: https://github.com/norwoodj/helm-docs/blob/v1.7.0/pkg/document/model.go#L14-L23
- Helm does not have a function to parse JSON

Reply:
So I could make it rendered like this, but then the lack of wrapping make it pretty weird:

> *Type:* `object`\
> *Default:*
> ```json
> {"authority":"https://login.microsoftonline.com/841df554-ef9d-48b1-bc6e-44cf8543a8fc/v2.0/.well-known/openid-configuration","autoUserInfo":false,"clientId":"01fcb3dc-7a2b-4b1c-a7d6-d7033089c779","ignoreNonceAfterRefresh":true,"issValidationOff":false,"logLevel":2,"maxIdTokenIatOffsetAllowedInSeconds":600,"postLogoutRedirectUri":null,"redirectUrl":null,"responseType":"id_token token","scope":"openid profile email offline_access api://wharf-internal/read api://wharf-internal/admin api://wharf-internal/deploy","silentRenew":true,"useRefreshToken":true}
> ```